core/src/main/java/com/ibm/watson/developer_cloud/service/WatsonService.java

```diff
@@ -97,6 +101,15 @@
    */
   public WatsonService(final String name) {
     this.name = name;
+    String iamApiKey = CredentialUtils.getIAMKey(name);
+    String iamUrl = CredentialUtils.getIAMUrl(name);
+    if (iamApiKey != null && iamUrl != null) {
```

Comment:
Even if the IAM API key is being pulled from VCAP credentials, the IAM URL is still optional, right? I think the `tokenManager` should still be created if there is an IAM API key but no IAM URL

Reply:
true

Reply:
@lpatino10 will you address this? As far as I can see, that is all that's left to be done

Reply:
Oops, I missed this one. Making the change now.